contrib/completion/bash/docker

```diff
@@ -1142,7 +1142,10 @@ __docker_complete_user_group() {
 	fi
 }
 
-DOCKER_PLUGINS_PATH=$(docker info --format '{{range .ClientInfo.Plugins}}{{.Path}}:{{end}}')
+__docker_plugins_path() {
+	DOCKER_PLUGINS_PATH=$(docker info --format '{{range .ClientInfo.Plugins}}{{.Path}}:{{end}}')
```

Comment:
`DOCKER_PLUGINS_PATH` very much looks like an environment variable, which is a bit confusing.
In the context of this refactoring, I'd prefer a lowercased name with `local` scope, e.g.

`local docker_plugins_path`

Reply:
OK